app/ide-desktop/lib/dashboard/src/layouts/PageSwitcher.tsx

```diff
@@ -24,28 +24,33 @@ export enum Page {
   settings = 'settings',
 }
 
-/** Error text for each page. */
-const ERRORS = {
-  [Page.drive]: null,
-  [Page.editor]: 'noProjectIsCurrentlyOpen',
-  [Page.settings]: null,
-} as const satisfies Record<Page, text.TextId | null>
+// =================
+// === Constants ===
+// =================
+
+/** Configuration flag determining whether to show the close button.
+ * This is present in order to match the Figma design as closely as possible. */
+const SHOW_CLOSE_BUTTON: boolean = false
+
+const PAGE_DATA: PageUIData[] = [
```

Comment:
I don't like this component is hardcoded for just 2 tabs.

This will require additional effort to refactor it to add multitab support. I thought that this is the goal of this PR: prepare the dashboard to multitab future

Reply:
It's not that easy already, because of some logic from the app-level leaked inside this component, for example, "Should it display the editor tab or not?"
It'd be better to extract that part outside of the "clean" part that is supposed only for rendering purposes. It should improve readability and further maintenance (look at the assets table). 

So my proposal is simple:

1."What to display" shall be separated from "How to display"
2. "How to display" shall be split into smaller, but reasonable parts, to improve readability and further maintenance.

It's much easier to read:

```
<Tabs>
 <Tabs.Tab />
 <Tabs.Tab />
</Tabs>
```

Than just their gust inside the single component